amethyst_renderer/src/pipe/stage.rs

```diff
@@ -190,6 +209,18 @@ where
         use std::cmp;
         cmp::max(L::len() * jobs_count, 1)
     }
+
+    fn new_targets(&mut self, new_targets: &HashMap<String, Target>) {
```

Comment:
doc?

Reply:
This is a trait impl, it's not needed here.